packages/yswaps/scripts/configs/mainnet.ts

```diff
@@ -47,10 +51,12 @@ const mainnetConfig: StrategyConfiguration<'MAINNET'> = {
           {
             tokenIn: '0xD533a949740bb3306d119CC777fa900bA034cd52', // CRV
             tokenOut: '0x29059568bB40344487d62f7450E78b8E6C74e0e5', // crvYfiEth
+            threshold: utils.parseEther('1'),
```

Comment:
If you could set this to a "40k threshold" in terms of CRV would be 10/10. Same with tokens above.